workflow-fingerprint-collector/src/main/java/ca/on/oicr/pde/workflows/FingerprintCollectorWorkflow.java

```diff
@@ -70,16 +59,17 @@
     private String bundledJRE;
     private String[] rgDetails;
     private boolean doBamFix = false;
+    private static final String INPUT_FILES="input_files";
 
     @Override
     public Map<String, SqwFile> setupFiles() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 37 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)